tests/lib/schema_validation_test.py

```diff
@@ -170,6 +171,30 @@ def test_valid_without_auto_neg_and_missing_speed(self, default_data):
 
         libnmstate.validator.validate(default_data)
 
+    def test_valid_with_sriov_total_vfs(self, default_data):
```

Comment:
Personally, I like testing the edge cases explicitly, which would be -1, 0, 256, 257 and maybe few extra values beside these. Since there are two categories, valid and invalid, it makes in my opinion sense to use `@pytest.parametrize` and one test for valid values and one for invalid values. What do you think?

Reply:
That seems better. Will do, thanks!

Reply:
Done!